identity.go

```diff
@@ -483,11 +482,34 @@ func LoadIdentity(cfg Config) (Identity, error) {
 	return id, nil
 }
 
-func getUniqueCerts(certs []*x509.Certificate, pool *CaPool) []*x509.Certificate {
+// getUniqueCerts will return a slice of unique certificates from the given slice
+func getUniqueCerts(certs []*x509.Certificate) []*x509.Certificate {
 	set := map[string]*x509.Certificate{}
 	var keys []string // track insertion order so that server certs come before pool certs
```

Comment:
does this rely on the calling function to sort the certs accordingly? it seems like it, should it rely on the caller doing that?

Reply:
It doesn't. That comment is copy pasta.  Will remove.